README.md

```diff
@@ -109,30 +109,63 @@ As this is the correct password, the user will be logged in. In the background,
 Keycloak.
 
 ## Launching and configuring the example
+
+You can launch the demo Keycloak service and an example legacy service using Docker Compose. 
+You can learn more about the legacy app in its [README.md](docker/legacy-system-example/README.md) file.
+
 1. Navigate to `./docker`
 2. Execute `docker-compose up`
-3. Open `http://localhost:8024/auth/admin/` in a browser
+3. Open [http://localhost:8024/auth/admin/](http://localhost:8024/auth/admin/) in a browser
 4. Log in with the credentials:
 * User: `admin`
 * Password: `admin`
-5. Navigate to "User federation":
+5. Navigate to "User federation" (e.g. for the `master` realm):
 
 ![Sidebar](readme-images/sidebar.png)
 
 6. Choose "User migration using a REST client" from the "Add provider..." dropdown:
 
 ![User federation dropdown](readme-images/user-federation.png)
 
-7. Provide the legacy system endpoint URI in the "Rest client URI" field:
+7. Provide the legacy system endpoint URI in the "Rest client URI" field.
+For the provided example legacy app the correct value is: `http://legacy-system-example:8080/user-migration-support`:
 
-![Rest client URI input](readme-images/field_rest_client_uri.png)
+![Rest client URI input for the provided example](readme-images/field-rest-client-uri-for-example-app.png)
 
 8. Click "save":
 
 ![Save button](readme-images/save_btn.png)
 
+9. Go to the [http://localhost:8024/auth/realms/master/account](http://localhost:8024/auth/realms/master/account) URI 
+(replace `master` if you enabled the plugin for a different realm). 
+Click the `Sign out` button to log out from the admin account and then the `Sign in` button to login as an example user:
```

Comment:
Move the `Sign out` to a different step, before going to the URI, as the user at this point is only signed into the `master` realm.